src/server/http/short_url_error.js

```diff
@@ -1,9 +1,8 @@
 import Boom from 'boom';
 
-export function handleShortUrlError(err) {
-  if (err.isBoom) return err;
-  if (err.status === 401) return Boom.unauthorized();
-  if (err.status === 403) return Boom.forbidden();
-  if (err.status === 404) return Boom.notFound();
-  return Boom.badImplementation();
+export function handleShortUrlError(error) {
+  if (!error.isBoom && !(error instanceof Error)) {
```

Comment:
I can do that, I was just worried about changing implementation details, not knowing if I would create a subtle bug somewhere.